cats-backend/src/modules/cats/cats/cats.controller.ts

```diff
@@ -8,12 +9,15 @@ export class CatsController {
   constructor(private readonly catsService: CatsService) {}
 
   @Get()
-  async findAll(): Promise<Cat[]> {
-    return this.catsService.findAll();
+  async getCats(
+    @Query('name') name?: string,
+    @Query('mouseName') mouseName?: string,
+  ): Promise<Cat[]> {
```

Comment:
```suggestion
  ) {
```
No need to state the type

Reply:
changed it